### PR TITLE
uploading /MI-10-MEAN/Pratiksha/PratikshaA3_1.html

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

File: Pratiksha/html/PratikshaA3_one.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <style>
+        .red{ color: red;}
+        .green{ color: green;}
+    </style>
+    
+</head>
+<body>
+    <h1>Ambersoft</h1>
+    <ol>
+        <li class="red">MI-MEAN
```
Comment (VIP5614) [inline Pratiksha/html/PratikshaA3_one.html:17]: li ki jagah ol ko type do aap

**Review by Snehapadvi96 — APPROVED** [review]

**Review by VIP5614 — APPROVED** [review]

File: Pratiksha/html/PratikshaA3_one.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                          <ul>
+                              <li type="disc">Anu</li>
+                              <li type="disc">Pihu</li>
+                          </ul>
+
+                       <li type="i">SYSTEM-2</li>
+                           <ul>
+                            <li type="disc">Anu</li>
+                            <li type="disc">Pihu</li>
+                           </ul>
```
Comment (Pawan1005) [inline Pratiksha/html/PratikshaA3_one.html:57]: remove extra line

File: Pratiksha/html/PratikshaA3_one.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+ 
+                   </ol> 
+                </li>  
+
```
Comment (Pawan1005) [inline Pratiksha/html/PratikshaA3_one.html:61]: remove extra lines

File: Pratiksha/html/PratikshaA3_one.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                            <li type="disc">Anu</li>
+                            <li type="disc">Pihu</li>
+                           </ul>
+ 
+                   </ol> 
+                </li>  
+                    
+    </ol>
+</body>
+</html>
```
Comment (Pawan1005) [inline Pratiksha/html/PratikshaA3_one.html:64]: add new line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
need indentation 
remove extra lines

**Pull request merged**